### PR TITLE
Migrate ca-summary, ready for review.

### PR DESCRIPTION
**Pull request by tapina**
Rebased Joe's branch and "finished" it off.

### CODE REVIEW

File: components/ca-summary.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+                    // increase col count so we can add an attribute to style it
+                    numOfCols++;
+                }
+
+                // keep a count of the number of items in this column
+                itemsPerCol++;
+            });
+
+            if (itemsPerCol === 1) numOfCols--; // We didn't add anything to the new column
```
Comment (ElementForge) [inline components/ca-summary.js]: Missing { } around single line if statement

File: components/ca-summary.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            // TODO: move this out into a generic function to reduce duplication in ca-list
+
+            schema = schema.properties || schema;
+
+            const results = [];
+            const sortedKeys = getSortedSchemaKeys(schema);
+
+            sortedKeys.forEach(key => {
+                // skip any properties we dont want to display - TODO use .filter()
+                if (key === 'links' || key.indexOf('$') !== -1) return;
```
Comment (ElementForge) [inline components/ca-summary.js]: Missing { } around single line if statement

File: examples/index.html [hunk 3]
```diff
@@ -24,6 +25,7 @@
       <li><a href="/ca-remote-image-capture"></a></li>
       <li><a href="/ca-select"></a></li>
       <li><a href="/ca-property"></a></li>
+      <li><a href="/ca-summary"></a></li>
```
Comment (ElementForge) [inline examples/index.html]: Why have ca-summary been added twice?

**Comment by tapina** [comment]
@ElementForge are you ok with this now?

**Comment by ElementForge** [comment]
Yep, good to go...

**Comment by tapina** [comment]
Needs another review too...

**Pull request merged**